src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/integration/cas1/planning/SpacePlannerTest.kt

```diff
@@ -25,7 +26,7 @@ class SpacePlannerTest : InitialiseDatabasePerClassTestBase() {
   lateinit var spacePlanner: SpacePlanner
 
   private fun findCharacteristic(propertyName: String) =
-    characteristicRepository.findByPropertyName(propertyName)!!
+    characteristicRepository.findByPropertyName(propertyName, ServiceName.approvedPremises)!!
```

Comment:
I think you have a build issue here (expecting string instead of enum)

Reply:
Yes I see it but the funny thing was not complaining about it in local it compile without errors